mne/viz/tests/test_3d.py

```diff
@@ -395,33 +395,34 @@ def test_snapshot_brain_montage():
 
 @testing.requires_testing_data
 @requires_nibabel()
+@requires_version('nilearn', '0.4')
 def test_plot_volume_source_estimates():
     """Test interactive plotting of volume source estimates."""
     forward = read_forward_solution(fwd_fname)
     sample_src = forward['src']
 
     vertices = [s['vertno'] for s in sample_src]
     n_verts = sum(len(v) for v in vertices)
-    n_time = 5
+    n_time = 2
     data = np.random.RandomState(0).rand(n_verts, n_time)
     vol_stc = VolSourceEstimate(data, vertices, 1, 1)
 
     for mode in ['glass_brain', 'stat_map']:
-        with pytest.warns(DeprecationWarning,
-                          match='scalars to be interpreted as an index'):
+        with pytest.warns(None):  # sometimes get scalars/index warning
```

Comment:
why did you change this? Wasn't it better to catch the exact warning in question?

Reply:
Tests failed as it's not emitted on all platforms (this is our standard way of handling these cases)